lists/transaction-types/README.md

```diff
@@ -0,0 +1,74 @@
+Transaction Type List
+=====================
+
+This is a list of existing, reserved and tentative
+[EIP-2718](https://eips.ethereum.org/EIPS/eip-2718) TransactionTypes.
+
+Transaction Types
+-----------------
+
+| Version Byte | Specs or Purpose |
+|--------------|------------------|
+| 0x00  | Reserved: indicates legacy (untyped) trancactions *(see notes below)* |
+| 0x01  | [EIP-2930](https://eips.ethereum.org/EIPS/eip-2930) *(available in Berlin)* |
+| 0x02  | [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559) *(available in London)* |
+| 0x03  | Reserved: prevents collision with [EIP-3074](https://eips.ethereum.org/EIPS/eip-3074) *(see notes below)* |
+| 0x18  | Reserved: prevents collision with [EIP-191](https://eips.ethereum.org/EIPS/eip-191) *(see notes below)* |
```

Comment:
Perhaps we can leave a general comment about "Seeing notes below" vs. having it in each row. 

Suggestion: add "For more details about each Transaction Type, see the "Reserved Transaction Types Motivation and History" section below." right above the table.

Reply:
Updated. :)